src/roswire/proxy/roslaunch/reader.py

```diff
@@ -23,10 +25,27 @@
 from ...name import (global_name, name_is_global, name_is_private, namespace,
                      namespace_join)
 
+
+class LaunchFileReader(abc.ABC):
+
+    @abc.abstractmethod
+    def read(self,
+             fn: str,
```

Comment:
`argv` should be keyword only, and `package` should be added as a keyword-only argument.

Reply:
Ignore this, I got confused with `LaunchFileManager` :facepalm: